### PR TITLE
Added fields for awardee + awarder karma

### PR DESCRIPTION
**Pull request by SupremeDeity**
Please review and merge. This adds the two fields required for getting all four types of karma.

### CODE REVIEW

File: lib/src/objector.dart [hunk 1]
```diff
@@ -93,11 +93,15 @@ class Objector extends RedditBase {
       return buildModeratorAction(reddit, data['data']);
     } else if (data.containsKey('sr') &&
         data.containsKey('comment_karma') &&
-        data.containsKey('link_karma')) {
+        data.containsKey('link_karma') &&
```
Comment (bkonyi) [inline lib/src/objector.dart:96]: This conditional seems to be for a `Subreddit` but awarder/awardee karma are `Redditor` properties. I think you can just drop the changes in this file and things should work as-is.

**Review by bkonyi — CHANGES_REQUESTED** [review]
Thanks for the contribution! Just one change and this should be good to go.

File: lib/src/objector.dart (hunk 1 above)
Comment (SupremeDeity) [reply lib/src/objector.dart]: Seems like i didn't pay attention to that at first. It should be resolved in the newest commit.

**Review by bkonyi — APPROVED** [review]

**Comment by bkonyi** [comment]
Thanks!

**Pull request merged**